app/assets/javascripts/pdf_viewer/base.js

```diff
@@ -0,0 +1 @@
+//= require_tree .
```

Comment:
Just checking does this not load everything in the JavaScript directory? 
If so is having it here different from having it in **app/assets/javascripts/hyku_addons.js**

Reply:
I will also double check this.

Reply:
As I understand it the docs say this will require all files in app/assets/javascripts/pdf_viewer and below: https://guides.rubyonrails.org/asset_pipeline.html#manifest-files-and-directives

Reply:
I have inspected the page and on the PDF Viewer only the JS and CSS from Hypothesis is being loaded:
![image](https://user-images.githubusercontent.com/7236943/145061762-bb06c36c-4353-4ae6-92fa-84944f3a8815.png)
On the main page, the PDF Viewer JS is not loaded:
![image](https://user-images.githubusercontent.com/7236943/145062050-ec5b4a05-a761-4bd3-9bb4-128550137dd4.png)